src/prisma/migrations/20250509130410_space_subscription/migration.sql

```diff
@@ -0,0 +1,53 @@
+-- AlterEnum
+-- This migration adds more than one value to an enum.
+-- With PostgreSQL versions 11 and earlier, this is not possible
+-- in a single migration. This can be worked around by creating
+-- multiple migrations, each migration adding only one value to
+-- the enum.
+
+
+ALTER TYPE "SubscriptionTier" ADD VALUE 'bronze';
```

Comment:
is this migration actually correct? it's extending the old subscription tier

Reply:
there are two migrations here. I will merge them into a single one